lib/util/fs/index.ts

```diff
@@ -176,6 +176,11 @@ export function createCacheWriteStream(path: string): fs.WriteStream {
   return fs.createWriteStream(fullPath);
 }
 
+export function createCacheReadStream(path: string): fs.ReadStream {
```

Comment:
please create a separate PR for this changes

Reply:
done, #30600